manifest.json

```diff
@@ -8,11 +8,11 @@
   "author": "Oscar Esteban, Krzysztof F. Gorgolewski. Poldrack Lab, Psychology, CRN, Stanford University",
   "cite": "Esteban O, Birman D, Schaer M, Koyejo OO, Poldrack RA, Gorgolewski KJ; MRIQC: Advancing the Automatic Prediction of Image Quality in MRI from Unseen Sites; PLOS ONE 12(9):e0184661; doi:10.1371/journal.pone.0184661.",
   "license": "BSD-3-Clause",
-  "version": "0.6.4_0.15.1",
+  "version": "0.6.5_0.15.1_dev5",
```

Comment:
I would bump this to 0.7.0 as we're adding a new feature (config option) with default settings that will result in a different output file than the previous version.